### DISCUSSION TITLE
Can you give the icons set ?

### DISCUSSION THREAD

**Discussion by Posandu** [blueedgetechno/win11React#55] [Q&A] [answered]
Can you give the icons set n windows 11? A ZIP file or RAR file is good. Thanks in advance.

**Comment by blueedgetechno** [comment, marked as answer]
go to /public/img/icon . Those are the only best I curated for the project.

**Reply by Posandu** [reply]
Thank you

**Reply by leasterli** [reply]
NB

**Comment by talha-mohamed** [comment]
thanks
